### ISSUE TITLE
No feedback associated with the event (IOS)

### ISSUE THREAD

**Issue by diasmalibekov** [getsentry/sentry-react-native#2788]
**OS:**
- [ ]  Windows
- [x]  MacOS
- [ ]  Linux

**Platform:**
- [x]  iOS
- [ ]  Android

**SDK:**
- [x]  `@sentry/react-native` (>= 1.0.0)
- [ ]  `react-native-sentry` (<= 0.43.2)

**SDK version:** 4.14.0

**`react-native` version:** 0.66.5

**Are you using Expo?**
- [ ]  Yes
- [x]  No

**Are you using sentry.io or on-premise?**
- [ ]  sentry.io (SaaS)
- [x]  on-premise

**If you are using sentry.io, please post a link to your issue so we can take a look:**

[Link to issue]

**Configuration:**

(`@sentry/react-native`)
```
Sentry.init({
    dsn: Config.SENTRY_DSN,
    integrations: [
      new Sentry.ReactNativeTracing({
        routingInstrumentation,
      }),
    ],
    beforeSend: (event, { captureContext } = {}) => {
      const { event_id } = event
      const { name, comments } = (captureContext as any)?.extra?.userFeedback ?? {}
      const { isFeedback } = (captureContext as any)?.tags ?? {}

      if (name && comments && event_id) {
        const userFeedback: Sentry.UserFeedback = {
          event_id,
          name,
          comments,
          email: '',
        }

        Sentry.captureUserFeedback(userFeedback)

        if (Platform.OS === 'ios') {
          Sentry.captureUserFeedback(userFeedback)
        }
      }

      if (isFeedback) {
        const changedEvent = JSON.parse(JSON.stringify(event))
        changedEvent.tags = { ...(event.tags ?? {}), isFeedback: true }

        return changedEvent
      }

      return event
    },
    tracesSampleRate: 1,
    // debug: __DEV__,
  })
```

[Description]

**Steps to reproduce:**
- Step 1: Sentry.captureMessage
- Step 2: beforeSend
- Step 3: inside beforeSend - Sentry.captureUserFeedback
  beforeSend because a lastEventId method returns 000000000(or smthng like this)

**Actual result:**
Sentry has the event, but has not a user feedback (ios). Android is OK!
But if I double the captureFeedBack method, it's working fine:
```
Sentry.captureUserFeedback(userFeedback)

if (Platform.OS === 'ios') {
  Sentry.captureUserFeedback(userFeedback)
}
```
**Expected result:**
Both of the userFeedBack and the event appears at Sentry.

**Comment by Hideart** [comment]
Same here
Looks so strange because Android working as expected

And I cannot re-send the feedback per same session at iOS. Sentry as if ignored my moves 🤷‍♂️

```
react-native: 0.66.1
@sentry/react-native: 4.10.0
```

**Comment by krystofwoldrich** [comment]
I've tried the `beforeSend` code from the sample, but I can't replicate the issue, if I call the `captureUserFeedback` only once on iOS I'm getting the event with user feedback.

`lastEventId` was added in [`4.12.0`](https://github.com/getsentry/sentry-react-native/releases/tag/4.12.0)

@Hideart could you check logs with `debug: true`? 

@diasmalibekov Could you send a minimal reproducible example to investigate this more?

**Comment by Hideart** [comment]
> I've tried the `beforeSend` code from the sample, but I can't replicate the issue, if I call the `captureUserFeedback` only once on iOS I'm getting the event with user feedback.
> 
> `lastEventId` was added in [`4.12.0`](https://github.com/getsentry/sentry-react-native/releases/tag/4.12.0)
> 
> @Hideart could you check logs with `debug: true`?
> 
> @diasmalibekov Could you send a minimal reproducible example to investigate this more?

Yeah, but its nothing about the feeadback

**Comment by Hideart** [comment]
@krystofwoldrich looks like the issue is with our on-premise sentry server. I'm not sure, but I'll check it and will write here

**Comment by krystofwoldrich** [comment]
Closing this due to inactivity. If this is still an issue feel free to comment here or open a new issue if the context changed.

**Comment by Hideart** [comment]
@krystofwoldrich Oh, I'm sorry dude. I was completely forget to write here.
The trouble relates to an our self-hosted instance
**Fixed it by updating to the latest**
